.github/workflows/push.yml

```diff
@@ -58,7 +58,9 @@ jobs:
           python-version: '3.9'
 
       - name: Install uv
-        uses: astral-sh/setup-uv@887a942a15af3a7626099df99e897a18d9e5ab3a # v5.1.0
+        uses: astral-sh/setup-uv@f94ec6bedd8674c4426838e6b50417d36b6ab231 # v5.3.1
```

Comment:
FYI, we have dependabot to update this once a month.

Reply:
You're right, something is off. It was working https://github.com/databricks/cli/pull/2116/files but no longer does. However, I do see dependabot doing upgrades (Feb 3, March 3).